analyzers/tests/SonarAnalyzer.UnitTest/TestCases/SymbolicExecution/Roslyn/NullPointerDereference.cs

```diff
@@ -1897,3 +1897,88 @@ public void WithNormalValue()
         }
     }
 }
+
+public class PeachReproducers
+{
+    public void ConditionalPropertyInt(string value)
+    {
+        if (value?.Length > 0)
+        {
+            value.ToString();   // Noncompliant
```

Comment:
Here and the follwing
```suggestion
            value.ToString();   // Noncompliant FP
```

Reply:
cc @Tim-Pohlmann: Don't trust me.. I forgot to put these "FP" comments all over the place 😅

Reply:
I noticed, but Martin had it pointed out already. I didn't think about the fact, that my approval would greenlight the PR for merging. Should I choose "Request Changes" and point to the unresolved comments in such a case?

Reply:
I saw the timeline and I supposed you know what you did. I just wanted to highlight it to be sure.

So as long as you trust me to address the comment, you did the right thing.